test/integration/integration.go

```diff
@@ -0,0 +1,9 @@
+package integration
+
+import "testing"
+
+func Skip(t testing.TB) {
```

Comment:
comments here just to explain what it's doing, and include how it works with run.go and skip.go